docs/resources/mcs_db_cluster.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: "mcs"
+page_title: "mcs: db_cluster"
+sidebar_current: "docs-db-cluster"
+subcategory: ""
+description: |-
+  Manages a db cluster.
+---
+
+# mcs\_db\_cluster (Resource)
+
+Provides a db cluster resource. This can be used to create, modify and delete db cluster for galera_mysql and postgresql datastores.
+
+## Example Usage
+
+```terraform
+
+resource "mcs_db_cluster" "mydb-cluster" {
+  name      = "mydb-cluster"
+  flavor_id = example_flavor_id
+  volume_size      = 10
+  volume_type = example_volume_type
+
+  cluster_size = 3
+
+  network {
+    uuid = example_network_id
+  }
+
+  datastore {
+    version = "12"
+    type    = "postgresql"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the cluster. Changing this creates a new cluster
+
+* `volume_size` - (Required) Size of the cluster instance volume.
+
+* `volume_type` - (Required) The type of the cluster instance volume. Changing this creates a new cluster.
+
+* `flavor_id` - (Required) The ID of flavor for the cluster.
+
+* `datastore` - (Required) Object that represents datastore of the cluster. Changing this creates a new cluster. It has following attributes:
+    * `type` - (Required) Type of the datastore. Changing this creates a new cluster. Type of the datastore can either be "galera_mysql" or "postgresql".
+    * `version` - (Required) Version of the datastore. Changing this creates a new cluster.
+
+* `cluster_size` - (Required) The number of instances in the cluster.
+
+* `network` -  Object that represents network of the cluster. Changing this creates a new cluster. It has following attributes: 
+    * `uuid` - The id of the network. Changing this creates a new cluster.
+    * `port` - The port of the network. Changing this creates a new cluster.
```

Comment:
The port id of the network. Add id to other such places as well

Reply:
Fixed